spynl/main/exceptions.py

```diff
@@ -16,9 +16,13 @@ class SpynlException(Exception):
     """
     http_escalate_as = HTTPBadRequest
 
-    def __init__(self, message='an internal error has occured', debug_message=None):
+    def __init__(self,
+                 message='an internal error has occured',
+                 debug_message=None,
+                 developer_message=None):
         self.message = message
```

Comment:
Sure but I would put them in the docstring of the __init__ method.
Other than that PR seems fine by me.